doc/radosgw/layout.rst

```diff
@@ -133,7 +133,8 @@ to how Extended Attributes associate with a POSIX file. An object's omap
 is not physically located in the object's storage, but its precise
 implementation is invisible and immaterial to RADOS Gateway.
 In Hammer, LevelDB is used to store omap data within each OSD; later releases
-default to RocksDB but can be configured to use LevelDB.
+default to RocksDB but can be configured to use LevelDB until LevelDB support
+has been removed in Quincy.
```

Comment:
since this documentation is going into main, i don't think it's useful to talk about hammer or quincy at all. can we just remove this text about leveldb/rocksdb? the sentence above says that this distinction isn't important anyway:
> An object's omap is not physically located in the object's storage, but its precise implementation is invisible and immaterial to RADOS Gateway.

Reply:
If it's transparent in the context of RGW, that makes sense.

Reply:
remove the text has done.